wandb/sdk/internal/internal_api.py

```diff
@@ -1404,32 +1420,79 @@ def create_run_queue(
             raise UnsupportedError(
                 "default resource configurations are not supported by this version of wandb server."
             )
+        if not supports_prioritization and prioritization_mode is not None:
+            raise UnsupportedError(
+                "launch prioritization is not supported by this version of wandb server."
+            )
 
-        query = gql(
-            """
-        mutation createRunQueue($entity: String!, $project: String!, $queueName: String!, $access: RunQueueAccessType!, $defaultResourceConfigID: ID) {
-            createRunQueue(
-                input: {
-                    entityName: $entity,
-                    projectName: $project,
-                    queueName: $queueName,
-                    access: $access,
-                    defaultResourceConfigID: $defaultResourceConfigID
+        if supports_prioritization:
+            query = gql(
+                """
+            mutation createRunQueue(
```

Comment:
matter of style, but maybe you can reduce duplication and just add the missing field in case `supports_prioritization` is true.